lcd/pyb_i2c_lcd_test.py

```diff
@@ -1,7 +1,7 @@
 """Implements a HD44780 character LCD connected via PCF8574 on I2C."""
 
 from pyb import I2C, delay, millis
-from lcd_api import LcdApi
```

Comment:
ok - this comment is still valid. but I see the next one is correcting an earlier deletion.